digitalidentity/share_session_created.go

```diff
@@ -0,0 +1,8 @@
+package digitalidentity
+
+// ShareSessionCreated Share Session QR Result
+type ShareSessionCreated struct {
```

Comment:
I don't know how GO architecture works but I'm pretty sure there is a way to reuse the response obj from the session creation

Reply:
there is, but the session object here is smaller than what's returned for session created - unless the api docs are mistaken?

Reply:
we have made this struct inline so i think this is now unnecessary

Reply:
dunno what API u're referring to but both `Session` creation and `QR` fetch response (limited to the sessione) are
```
{
  "id": "",
  "status": "",
  "expiry": ""
}

Reply:
when you fetch the session, the [session resource](https://github.com/getyoti/yoti-go-sdk/blob/SDK-2259-go-retrieve-qr-code/digitalidentity/share_session.go#L4-L12) seems to be larger than what is returned in the `fetch qr code` response (what you posted), we could embed it and just have it unmarshal the whole thing, but some extra fields will just be empty for the RP, imo it's better to avoid it if the API itself specifically limits the fields and not give the RP any cause for confusion or incorrect expectations.

Reply:
sorry I thought I answered this but I must have not pushed the button. I meant this [session created response](https://github.com/getyoti/yoti-go-sdk/blob/SDK-2259-go-retrieve-qr-code/digitalidentity/share_session_created.go)